cypress/e2e/20-workflow-executions.cy.ts

```diff
@@ -183,6 +183,50 @@ describe('Current Workflow Executions', () => {
 			.invoke('attr', 'title')
 			.should('eq', newWorkflowName);
 	});
+
+	it('should load items and auto scroll after filter change', () => {
+		createMockExecutions();
```

Comment:
Can we refactor this so it takes execution number(s) as parameter? This way we won't have to call it multiple times

Reply:
This function creates 11 executions in one go, 9 successful and 2 failed in the following order: 5 success, 2 error, 4 success

I just needed more executions to make the list scrollable, so I called it twice.

There would be so many ways to parameterize this but I think it could be overcomplicated.
If it was just a number, that's fine, but there are more here.
Like how many successful and failed executions you want to generate (that's still just two numbers, fine) but in what order would make it not worth doing

If you want more fine-grained control, you could just call the helper functions that are called inside that function.